app/views/application_drafts/index.html.slim

```diff
@@ -18,6 +19,12 @@ table.table.table-striped.table-bordered.table-condensed
   tbody
     - @application_drafts.each do |draft|
       tr
+        td
+          - if draft.position == 1
+            i.icon-star title='Preferred application'
+          - else
+            a href=prioritize_application_draft_path(draft) title='Make preferred' data-method='put'
```

Comment:
Nah, it the jquery-ujs does a post rather than a xhr request, the controller then redirects as normal. So the change will be immediately be visible.